cmd/shipperctl/cmd/apply.go

```diff
@@ -94,6 +99,20 @@ func runApplyClustersCommand(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func validateConfiguration(configuration *config.ClustersConfiguration) error {
```

Comment:
Consider a method name reflecting cluster-bounded scope of the operation, e.g.: `validateClusterConfiguration`